### PR TITLE
ENH Add CUDA 11.2 support

### PR DESCRIPTION
**Pull request by mike-wendt**
Updating axis files to build meta/env packages for CUDA 11.2 in order to support CUDA 11.2 builds and testing.

This is related to work in rapidsai/gpuci-build-environment#167 to add CUDA 11.2 support to gpuCI images.

### CODE REVIEW

**Comment by mike-wendt** [comment]
Need CUDA 11.2 images which were just merged in rapidsai/gpuci-build-environment#167 and should build momentarily, will trigger a retest once the images are available

**Comment by mike-wendt** [comment]
Verified new images are [built](https://gpuci.gpuopenanalytics.com/job/gpuci/job/gpuci-build-environment-jobs/job/miniconda-cuda-driver/52/)

rerun tests

**Comment by mike-wendt** [comment]
`gpuCI/build/meta-pkg-nightly` will have expected failures for CUDA 11.2 as we do not publish any RAPIDS libraries for CUDA 11.2 yet

We need to see `gpuCI/build/env-pkg-nightly` pass to unblock the `gpuci/rapidsai` and `gpuci/rapidsai-driver` images from being built in an upcoming PR in rapidsai/gpuci-build-environment

**Comment by mike-wendt** [comment]
Looks like nccl needs an update for CUDA 11.2, investigating that

**Comment by mike-wendt** [comment]
`gpuCI/build/rapids-integration-tests` failed on CUDA 10.1 due to an [abort](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/integration/job/prb/job/rapids-integration-tests/283/) the issue there is being addressed and should not block this PR

**Review by ajschmidt8 — APPROVED** [review]

**Review by raydouglass — APPROVED** [review]

**Review by kkraus14 — APPROVED** [review]

**Pull request merged**
